esp/src/src/nls/bs/hpcc.ts

```diff
@@ -125,13 +130,17 @@
     ContinueWorking: "Nastavite sa radom",
     Copied: "Kopirano!",
     Copy: "Kopirajte",
+    CopyLogicalFilename: "Kopirajte naziv logičke datoteke",
     CopyLogicalFiles: "Kopirajte logičke fajlove u klipbord",
+    CopySelectionToClipboard: "Kopiraj odabir u međuspremnik",
```

Comment:
"Kopiraj odabir u klipbord"

for consistency with previous

Reply:
updated for consistency